lib/build.py

```diff
@@ -92,6 +88,28 @@ def main(cli_):
       cli.tag = re.sub(r"[^a-z0-9_.-]", "", cli.tag.lower())
       ch.INFO("inferred image name: %s" % cli.tag)
 
+   # --force and friends.
+   if (cli.force_cmd and cli.force == "fakeroot"):
+      ch.FATAL("--force-cmd and --force=fakeroot are incompatible")
+   if (not cli.force_cmd):
+      cli.force_cmd = force.FORCE_CMD_DEFAULT
+   else:
+      cli.force = "seccomp"
+      # convert cli.force_cmd to parsed dict
+      force_cmd = dict()
+      for line in cli.force_cmd:
+         (cmd, args) = force.force_cmd_parse(line)
+         force_cmd[cmd] = args
+      cli.force_cmd = force_cmd
+   ch.VERBOSE("force mode: %s" % cli.force)
+   if (cli.force == "seccomp"):
+      for (cmd, args) in cli.force_cmd.items():
+         ch.VERBOSE("force command: %s" % ch.argv_to_string([cmd] + args))
+   if (    cli.force == "seccomp"
+       and ch.cmd([ch.CH_BIN + "/ch-run", "--feature=seccomp"],
```

Comment:
Okay, after a dozen or so read-throughs and some sleep, I think I see it now -- `cli.force` is preceded by spaces so that it lines up with `ch.cmd` and leaves room for the `and`, yes?  I'm not sure exactly why it threw me off nor quite how to eliminate the problem (that I've yet to identify...).  I do it differently, so maybe it's just my being unaccustomed to that style....

The only thing coming to mind is to maybe put a space before `ch.cmd` so that the `.` in `cli.` lines up with the one in `ch.`...maybe?  🤷